### DISCUSSION TITLE
zero replicas vs Degraded status

### DISCUSSION THREAD

**Discussion by aljesusg** [kiali/kiali#3887] [General]
Should we show a degraded status if the user scale down the app ?

![Screenshot from 2021-04-15 10-26-24](https://user-images.githubusercontent.com/3019213/114838528-13294a00-9dd5-11eb-83d5-886eb969072f.png)

I think that Iddle status should have more priority despite the last X minutes the health was degraded or failure. 

What do you think?

**Comment by lucasponce** [comment]
I think that's due it's taking the 1m telemetry info just when the resource is scale down.
Once that workload stops receiving telemetry it will show the "not ready".

To me it's fine to keep it as it's as it tells user (it has traffic in the past - 1 minute ago - but now you won't get more traffic).

Note that also we are "tweaking" how to present this info to the user https://github.com/kiali/kiali-ui/pull/2122, but the detail page will be clear in that case (clear that now it's scale down but it has previous telemetry in the interval).

**Comment by aljesusg** [comment]
> I think that's due it's taking the 1m telemetry info just when the resource is scale down.
> Once that workload stops receiving telemetry it will show the "not ready".
> 
> To me it's fine to keep it as it's as it tells user (it has traffic in the past - 1 minute ago - but now you won't get more traffic).
> 
> Note that also we are "tweaking" how to present this info to the user [kiali/kiali-ui#2122](https://github.com/kiali/kiali-ui/pull/2122), but the detail page will be clear in that case (clear that now it's scale down but it has previous telemetry in the interval).

yes I agree the traffic by default is the last minute but maybe this could be confuse from user . is aw this checking another issue and I wanted to open a discussion about this
